apisix/plugins/traffic-split.lua

```diff
@@ -320,6 +324,7 @@ function _M.access(conf, ctx)
         core.log.info("upstream: ", core.json.encode(upstream))
         return set_upstream(upstream, ctx)
     elseif upstream and upstream ~= "plugin#upstream#is#empty" then
+        ctx.matched_route.value["original_uid"] = ctx.matched_route.value.upstream_id
```

Comment:
Should do the same thing with the code below?
```
    core.log.info("route_up: ", upstream)
    ctx.matched_route.value.upstream_id = nil
```

Reply:
I don't think so, because here it will directly return
https://github.com/apache/apisix/blob/4820ad93e5fc0550addbd7029a20eab6b7ea800e/apisix/plugins/traffic-split.lua#L322-L325

Reply:
I mean, if upstream == "plugin#upstream#is#empty", will the original upstream_id be overridden?

Reply:
yes, it will be overridden, I continue to fix it.